src/components/inputs/async-select-input/async-select-input.js

```diff
@@ -123,12 +126,15 @@ export class AsyncSelectInput extends React.Component {
               ...customizedComponents,
               ...this.props.components,
             }}
-            styles={createSelectStyles({
-              hasWarning: this.props.hasWarning,
-              hasError: this.props.hasError,
-              showOptionGroupDivider: this.props.showOptionGroupDivider,
-              menuPortalZIndex: this.props.menuPortalZIndex,
-            })}
+            styles={createSelectStyles(
```

Comment:
don't have access to emotion here, so don't have the `theme` object without injecting it